fsdriver/driver_basic_darwin.go

```diff
@@ -0,0 +1,11 @@
+package fsdriver
+
```

Comment:
This should compile on darwin. Please don't remove support. This was probably already working, so adding build tags will break things. Should probably be called `_unix.go` or something like that.